src/main/java/uk/nhs/tis/trainee/notifications/dto/HistoryDto.java

```diff
@@ -24,17 +24,21 @@
 import java.time.Instant;
 import uk.nhs.tis.trainee.notifications.model.MessageType;
 import uk.nhs.tis.trainee.notifications.model.NotificationType;
+import uk.nhs.tis.trainee.notifications.model.TisReferenceType;
 
 /**
  * A DTO for historical notification data.
  *
- * @param id      The ID of the notification.
- * @param type    The type of notification e.g. EMAIL
- * @param subject The subject of the notification e.g. COJ_CONFIRMATION
- * @param contact The contact details used to send the notifiation.
- * @param sentAt  The timestamp the notification was sent at.
+ * @param id               The ID of the notification.
+ * @param tisReferenceType The TIS reference type for the entity that prompted the notification.
+ * @param tisReferenceId   The TIS ID of the entity that prompted the notification.
+ * @param type             The type of notification e.g. EMAIL
+ * @param subject          The subject of the notification e.g. COJ_CONFIRMATION
+ * @param contact          The contact details used to send the notification.
+ * @param sentAt           The timestamp the notification was sent at.
  */
-public record HistoryDto(String id, MessageType type, NotificationType subject, String contact,
+public record HistoryDto(String id, TisReferenceType tisReferenceType, String tisReferenceId,
```

Comment:
It would be cleaner to use `TisReferenceInfo` here instead of adding two fields, if we really need to we can use `@JsonUnwrapped` but I don't think it matter's if its a sub-object?

Reply:
Aw, I liked that it was a nice simple flat object 😁 .

Ok, let me make the change.

Reply:
Ah, wait. I thought it was already using some nested objects 🤔 
I think it still makes sense to group those fields together, but leave it flat if you want.